src/components/CompetenceGoalTab.tsx

```diff
@@ -52,19 +45,19 @@ export interface CoreElementType {
 }
 
 type CompetenceType = "goal" | "element";
-
+// TODO: adjust according to new design
```

Comment:
Her er du allerede ganske godt på vei. Kanskje bare fullføre jobben?

Reply:
Men det er jo ikke en del av søkesiden, derfor har jeg lyst til å gjøre det for seg, tar det med en gang denne her er inne